.github/workflows/integration.yml

```diff
@@ -30,15 +28,18 @@ env:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
What's the reason for this change? As long as there's no incompatibilities with `ubuntu-latest` I'd prefer to keep it to not have to manually bump it later. But no hard feelings here :shrug:

Reply:
I prefer following the principle of having fixed versions instead of floating ones. This way, I aim to avoid any surprises in the future. 😄